acdc-ws/app/controllers/StatusController.scala

```diff
@@ -8,17 +8,35 @@
 package controllers
 
 import javax.inject._
-
 import play.api.libs.json._
 import play.api.mvc._
-
 import com.salesforce.mce.acdc.ws.BuildInfo
+import utils.Metrics.{apiLatencyGauge, apiLatencySummary, clearMetrics, setSumAvgToGauge}
+import utils.{MetricReporter, ProfileAction}
+
+import scala.concurrent.ExecutionContext
 
 @Singleton
-class StatusController @Inject() (cc: ControllerComponents) extends AbstractController(cc) {
+class StatusController @Inject() (
+  cc: ControllerComponents,
+  reporter: MetricReporter
+) (implicit
+   ec: ExecutionContext
+)  extends AbstractController(cc) {
+
+  def status: Action[AnyContent] = ProfileAction(reporter) {
```

Comment:
I think we may not want to profile the status / metrics Action.  They are kinda like background noise wrt to the actual acdc http actions.

Reply:
Agree.